ducktape/command_line/main.py

```diff
@@ -130,6 +132,16 @@ def main():
         print ducktape_version()
         sys.exit(0)
 
+    injected_args = None
+    if args.inject:
+        try:
+            injected = args.inject
+            injected = injected.replace("'", "\"")
```

Comment:
I forgot to note before that I found this escaping confusing. Your example above clarifies why you did it though. You gave this as an example:

`ducktape test_something --args "{'x': 100, 'y': 15}"`

If you write it as 

`ducktape test_something --args '{"x": 100, "y": 15}'`

instead, then you don't need this escaping. Even if you didn't want to use the single quotes (which would, e.g., defeat variable substitution), you could write:

`ducktape test_something --args "{\"x\": 100, \"y\": 15}"`

I think we should avoid doing this escaping ourselves -- if we do it, then people trying to do valid things might get unexpected results. For example

`ducktape test_something --args "{\"param\": \"string value with aposrophe: it's\" }"`

will have the `'` character replaced incorrectly.